plugins/woocommerce/src/Blocks/BlockTypes/ProductGallery.php

```diff
@@ -77,25 +77,27 @@ function( $carry, $item ) {
 
 		$gallery_dialog = strtr(
 			'
-		<div class="wc-block-product-gallery-dialog__overlay" hidden data-wc-bind--hidden="!context.isDialogOpen" data-wc-watch="callbacks.keyboardAccess">
-			<dialog data-wc-bind--open="context.isDialogOpen">
-			<div class="wc-block-product-gallery-dialog__header">
-			<div class="wc-block-product-galler-dialog__header-right">
-				<button class="wc-block-product-gallery-dialog__close" data-wc-on--click="actions.closeDialog">
-					<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-						<rect width="24" height="24" rx="2"/>
-						<path d="M13 11.8L19.1 5.5L18.1 4.5L12 10.7L5.9 4.5L4.9 5.5L11 11.8L4.5 18.5L5.5 19.5L12 12.9L18.5 19.5L19.5 18.5L13 11.8Z" fill="black"/>
-					</svg>
-				</button>
-			</div>
-			</div>
-			<div class="wc-block-product-gallery-dialog__body">
-				{{html}}
-			</div>
+		<div class="wc-block-product-gallery-dialog__overlay" hidden data-wc-bind--hidden="!context.isDialogOpen" data-wc-watch="callbacks.keyboardAccess" data-wc-watch--check-dialog-visibility="callbacks.checkDialogVisibility">
```

Comment:
I've searched through the codebase and it seems that `callbacks.checkDialogVisibility` doesn't exist. Should we consider removing this attribute, or is there something I might be overlooking?

Reply:
Nice catch! I had initially added this, but in the end I decided to use another logic that would reduce complexity.

Reply:
I just removed this part in https://github.com/woocommerce/woocommerce/pull/44332/commits/e9e429283901158bb6576622f69372c056ac847c